<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="pinout_label">
      <property name="text">
       <string>Pinout</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pinout_text_edit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="processors_label">
      <property name="text">
       <string>Processor(s)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="processors_text_edit"/>
    </item>
    <item>
     <widget class="QLabel" name="results_label">
      <property name="text">
       <string>Results</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="results_table_widget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>128</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>RPy</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ANn</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CNm</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Save</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
